paimon-web-common/pom.xml

```diff
@@ -38,11 +38,6 @@ under the License.
     </properties>
 
     <dependencies>
-        <dependency>
```

Comment:
Why remove this dependency? Is this change related with job submission?

Reply:
Thank you for bringing this to my attention. You're correct; this part isn't related to the current PR. I will take the revert it.

Reply:
Thank you for bringing this to my attention. You're correct; this part isn't related to the current PR. I will take the revert it.

Reply:
Thank you for bringing this to my attention. You're correct; this part isn't related to the current PR. I will take the revert it.